.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -40,15 +40,17 @@ assignees: ''
 
 **After release**:
 
+- [ ] create branch 6.x for bug-fixes if this is a minor release (i.e. create branch 6.1 after 6.1.0 is released)
 - [ ] update `GMT_PACKAGE_VERSION_*` in `cmake/ConfigDefault.cmake`
-- [ ] set `GMT_PUBLIC_RELEASE` to `FALSE`
+- [ ] comment the `set (GMT_PUBLIC_RELEASE TRUE)` line
+- [ ] commit changes to GitHub
 
 **3rd-party update**
 
 - [ ] update [conda-forge feedstock](https://github.com/conda-forge/gmt-feedstock) (@leouieda, @seisman)
 - [ ] update [homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gmt.rb) (@seisman)
```

Comment:
@claudiodsf Are you willing to help maintain the updates of the homebrew formula? If yes, I can add you to the checklist.

Reply:
Sure! Sorry for late reply.